plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeFileScanTest.kt

```diff
@@ -131,6 +142,33 @@ class CodeWhispererCodeFileScanTest : CodeWhispererCodeScanTestBase(PythonCodeIn
         virtualFile4 = psifile4.virtualFile
         file4 = virtualFile4.toNioPath().toFile()
 
+        // Create a 200KB file
+        val content = "a".repeat(200 * 1024)
+        psifilePerformanceTest = projectRule.fixture.addFileToProject("test.txt", content)
+        performanceTestfileWithPayload200KB = psifilePerformanceTest.virtualFile.toNioPath().toFile()
+
+        sessionConfigSpy3 = spy(
+            CodeScanSessionConfig.create(
+                psifilePerformanceTest.virtualFile,
+                project,
+                CodeWhispererConstants.CodeAnalysisScope.FILE
+            )
+        )
+        setupResponse(psifilePerformanceTest.virtualFile.toNioPath().relativeTo(sessionConfigSpy3.projectRoot.toNioPath()))
+
+        // Create a 150KB file
+        val codeContentForPayload = "a".repeat(150 * 1024)
+        psifilePerformanceTest2 = projectRule.fixture.addFileToProject("test.txt", codeContentForPayload)
+        performanceTestfileWithPayload150KB = psifilePerformanceTest2.virtualFile.toNioPath().toFile()
+
+        sessionConfigSpy4 = spy(
+            CodeScanSessionConfig.create(
+                psifilePerformanceTest2.virtualFile,
+                project,
+                CodeWhispererConstants.CodeAnalysisScope.FILE
+            )
+        )
+        setupResponse(psifilePerformanceTest2.virtualFile.toNioPath().relativeTo(sessionConfigSpy4.projectRoot.toNioPath()))
```

Comment:
does this need to be in the global setup method? it is only ever used in a single test